### DISCUSSION TITLE
readonly cluster shows broker config edit button

### DISCUSSION THREAD

**Discussion by senthil13** [provectus/kafka-ui#3086] [Q&A] [answered]
Hi Team - Is this working as expected? I set up a read-only cluster, but when I navigate to the broker config page, it shows the edit button. when I try to edit and apply its showing throwing error 405 Method Not Allowed. But it shouldn't be showing the edit button at all right? considering this is a read-only cluster?

Thanks

**Comment by Haarolean** [comment, marked as answer]
Hey, currently that's an expected behaviour. We're gonna do this the proper way within #753.
